extras/basic_example/public/index.html

```diff
@@ -1,7 +1,7 @@
 <html>
   <head>
     <title>Licode Basic Example</title>
-    <script type="text/javascript" src="erizo.js"></script>
+    <script type="text/javascript" src="http://192.168.33.99:8080/erizo.js"></script>
```

Comment:
Oh, I remember now.  I've made a note in the wiki to help remind me the next time I come across that IP.